data/human/names.txt

```diff
@@ -3318,8 +3334,11 @@ phrase "intellectuals"
 		"Fourier"
 		"Franklin"
 		"Fresnel"
+		"Galileo"
```

Comment:
As this was already a point of contention, I'm fine with this not being changed. However, I personally think this should also be a last name, for consistency.